src/ATen/native/xpu/GRUFusedCell.cpp

```diff
@@ -0,0 +1,41 @@
+#include <ATen/ATen.h>
+#include <ATen/core/op_registration/adaption.h>
+#include <ATen/native/cpu/mixed_data_type.h>
+#include <ATen/native/xpu/sycl/GRUFusedCellKernels.h>
+#include <ATen/xpu/XPUNativeFunctions.h>
```

Comment:
This header file seems to be non-existent from the build logs.
Would changing to `XPUFunctions.h` instead of `XPUNativeFunctions.h` make it work?